mapillary_tools/processing.py

```diff
@@ -210,7 +241,7 @@ def geotag_from_gps_trace(process_file_list,
 
     if use_gps_start_time:
         # update offset time with the gps start time
-        offset_time += (sub_second_times[0] - gps_trace[0][0]).total_seconds()
+        offset_time += (sorted(sub_second_times)[0] - gps_trace[0][0]).total_seconds()
```

Comment:
yes i think you r right, this should be sorted to take the smallest time, i will add it to master already since it could result in jumps otherwise